beamer/tests/agent/test_handlers.py

```diff
@@ -169,3 +175,32 @@ def test_request_garbage_collection_with_claim(token, config: Config):
     process_requests(context)
     assert len(context.requests) == 0
     assert len(context.claims) == 0
+
+
+def test_handle_initiate_l1_resolution(token: Contract, config: Config):
+    request = make_request(token)
+
+    context = make_context(config)
+    context.requests.add(request.id, request)
+
+    event = InitiateL1ResolutionEvent(
+        chain_id=TARGET_CHAIN_ID,
+        request_id=REQUEST_ID,
+        claim_id=CLAIM_ID,
+    )
+
+    # Without a claim, this must fail
+    assert process_event(event, context) == (False, None)
+
+    claim = make_claim(request, claimer=config.account.address)
+    context.claims.add(claim.id, claim)
+
+    # Must only be called if request is filled
+    with pytest.raises(AssertionError, match="Request not yet filled"):
+        process_event(event, context)
+
+    # Check that task is added to resolution pool
+    context.web3_l1.eth.gas_price = 1
```

Comment:
Why do we change gas price here?
Would this change persist across all later tests?